Misc/NEWS

```diff
@@ -688,6 +688,9 @@ Library
 - Issue #24142: Reading a corrupt config file left configparser in an
   invalid state.  Original patch by Florian Höch.
 
+- Issue #29271: Fix Task.current_task and Task.all_tasks to accept
```

Comment:
Move the entry to the start of the section. Mention that this fixes C implementation, makes it conforming to Python implementation.